[2025-06-09T10:29:10.341+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_ecommerce_pipeline.check_services_health manual__2025-06-09T10:25:24.080349+00:00 [queued]>
[2025-06-09T10:29:10.346+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_ecommerce_pipeline.check_services_health manual__2025-06-09T10:25:24.080349+00:00 [queued]>
[2025-06-09T10:29:10.346+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-06-09T10:29:10.353+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): check_services_health> on 2025-06-09 10:25:24.080349+00:00
[2025-06-09T10:29:10.356+0000] {standard_task_runner.py:60} INFO - Started process 271 to run task
[2025-06-09T10:29:10.361+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'kafka_ecommerce_pipeline', 'check_services_health', 'manual__2025-06-09T10:25:24.080349+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/kafka_data_pipeline.py', '--cfg-path', '/tmp/tmp4umhsxhu']
[2025-06-09T10:29:10.365+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask check_services_health
[2025-06-09T10:29:10.396+0000] {task_command.py:423} INFO - Running <TaskInstance: kafka_ecommerce_pipeline.check_services_health manual__2025-06-09T10:25:24.080349+00:00 [running]> on host 426c4f7e0cdd
[2025-06-09T10:29:10.436+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data-engineering-team' AIRFLOW_CTX_DAG_ID='kafka_ecommerce_pipeline' AIRFLOW_CTX_TASK_ID='check_services_health' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T10:25:24.080349+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T10:25:24.080349+00:00'
[2025-06-09T10:29:10.437+0000] {kafka_data_pipeline.py:55} INFO - üîç Checking service health...
[2025-06-09T10:29:10.438+0000] {kafka_data_pipeline.py:61} INFO - ‚úÖ Redis is reachable at redis:6379
[2025-06-09T10:29:10.440+0000] {conn.py:396} INFO - <BrokerConnection client_id=airflow_health_check, node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.18.0.5', 29092)]>: connecting to kafka:29092 [('172.18.0.5', 29092) IPv4]
[2025-06-09T10:29:10.446+0000] {conn.py:618} INFO - <BrokerConnection client_id=airflow_health_check, node_id=bootstrap-0 host=kafka:29092 <checking_api_versions_recv> [IPv4 ('172.18.0.5', 29092)]>: Broker version identified as 2.6
[2025-06-09T10:29:10.447+0000] {conn.py:457} INFO - <BrokerConnection client_id=airflow_health_check, node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.18.0.5', 29092)]>: Connection complete.
[2025-06-09T10:29:10.453+0000] {conn.py:396} INFO - <BrokerConnection client_id=airflow_health_check, node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.18.0.5', 29092)]>: connecting to kafka:29092 [('172.18.0.5', 29092) IPv4]
[2025-06-09T10:29:10.453+0000] {conn.py:457} INFO - <BrokerConnection client_id=airflow_health_check, node_id=1 host=kafka:29092 <connected> [IPv4 ('172.18.0.5', 29092)]>: Connection complete.
[2025-06-09T10:29:10.454+0000] {conn.py:945} INFO - <BrokerConnection client_id=airflow_health_check, node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.18.0.5', 29092)]>: Closing connection. 
[2025-06-09T10:29:10.461+0000] {kafka_data_pipeline.py:80} INFO - ‚úÖ Kafka is reachable at kafka:29092
[2025-06-09T10:29:10.461+0000] {conn.py:945} INFO - <BrokerConnection client_id=airflow_health_check, node_id=1 host=kafka:29092 <connected> [IPv4 ('172.18.0.5', 29092)]>: Closing connection. 
[2025-06-09T10:29:10.461+0000] {kafka_data_pipeline.py:91} INFO - ‚ÑπÔ∏è KafkaAdminClient closed.
[2025-06-09T10:29:10.461+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-09T10:29:10.469+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=kafka_ecommerce_pipeline, task_id=check_services_health, execution_date=20250609T102524, start_date=20250609T102910, end_date=20250609T102910
[2025-06-09T10:29:10.509+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-09T10:29:10.530+0000] {taskinstance.py:3280} INFO - 2 downstream tasks scheduled from follow-on schedule check
